coala_quickstart/generation/Utilities.py

```diff
@@ -0,0 +1,14 @@
+from coalib.parsing.StringProcessing import unescaped_split
+
+
+def comma_split(response):
+    """
+    Splits the comma-separated string into a list. This takes escapes into
```

Comment:
I'm basically using the library available in coala `coalib.parsing.StringProcessing.unescaped_split`